core/src/visitors.rs

```diff
@@ -266,44 +279,50 @@ impl<'ast, 'a> Visit<'ast> for TypeShareVisitor<'a> {
 
     /// Collect rust structs.
     fn visit_item_struct(&mut self, i: &'ast syn::ItemStruct) {
-        if has_typeshare_annotation(&i.attrs) && !self.target_os_skipped(&i.attrs) {
-            self.collect_result(parse_struct(i, self.target_os.as_deref()));
+        debug!("Visiting {}", i.ident);
+        if has_typeshare_annotation(&i.attrs) && self.target_os_accepted(&i.attrs) {
+            debug!("\tParsing {}", i.ident);
+            self.collect_result(parse_struct(i, self.target_os));
         }
 
         syn::visit::visit_item_struct(self, i);
     }
 
     /// Collect rust enums.
     fn visit_item_enum(&mut self, i: &'ast syn::ItemEnum) {
-        if has_typeshare_annotation(&i.attrs) && !self.target_os_skipped(&i.attrs) {
-            self.collect_result(parse_enum(i, self.target_os.as_deref()));
+        debug!("Visiting {}", i.ident);
+        if has_typeshare_annotation(&i.attrs) && self.target_os_accepted(&i.attrs) {
+            debug!("\tParsing {}", i.ident);
+            self.collect_result(parse_enum(i, self.target_os));
         }
 
         syn::visit::visit_item_enum(self, i);
     }
 
     /// Collect rust type aliases.
     fn visit_item_type(&mut self, i: &'ast syn::ItemType) {
-        if has_typeshare_annotation(&i.attrs) && !self.target_os_skipped(&i.attrs) {
+        debug!("Visiting {}", i.ident);
+        if has_typeshare_annotation(&i.attrs) && self.target_os_accepted(&i.attrs) {
+            debug!("\tParsing {}", i.ident);
             self.collect_result(parse_type_alias(i));
         }
 
         syn::visit::visit_item_type(self, i);
     }
 
     /// Track potentially skipped modules.
-    fn visit_item_mod(&mut self, i: &'ast syn::ItemMod) {
-        if let Some(target_os) = self.target_os.as_ref() {
-            if i.attrs.iter().any(|attr| target_os_skip(attr, target_os)) {
-                println!("skip {}", i.ident);
-            }
-        };
+    // fn visit_item_mod(&mut self, i: &'ast syn::ItemMod) {
+    //     if let Some(target_os) = self.target_os.as_ref() {
+    //         if i.attrs.iter().any(|attr| target_os_skip(attr, target_os)) {
+    //             debug!("skip {}", i.ident);
+    //         }
+    //     };
 
-        syn::visit::visit_item_mod(self, i);
-    }
+    //     syn::visit::visit_item_mod(self, i);
+    // }
```

Comment:
leftover commented code

Reply:
It was there before but not really used. In the future I want to handle `#[cfg(target_os = "target")]` on `mod` definitions. It doesn't do that now but I'd like to add that later on.